<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Hotkey Recorder Widget Implementation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayoutHotkeyRecorder">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutHotkeyRecorderRecordedHotkey">
        <item>
         <widget class="QLineEdit" name="lineEditRecordedHotkeyVkid">
          <property name="placeholderText">
           <string>Recorded Hotkey VKID</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
